config/deploy.rb

```diff
@@ -84,6 +84,15 @@
     end
   end
 
+  desc "Deploy Google Cloud Function"
+  task :google_cloud_function do
+    on roles(:db) do
+      run_locally do
+        execute "cd cloud_fixity && gcloud functions deploy checkFixity --runtime nodejs8 --set-env-vars BUCKET=#{fetch(:google_fixity_bucket)},FIXITY_STATUS_TOPIC=#{fetch(:google_fixity_status_topic)} --trigger-topic #{fetch(:google_fixity_request_topic)} --service-account #{fetch(:google_service_account)}"
```

Comment:
`gcloud` is probably the preferred approach, but I would be curious as to whether or not it is at all possible to integrate something like Grunt or Gulp in order to wrap these types of tasks.

Reply:
I dunno! I'm not too psyched about the idea of adding another tool to our deployment pipeline though.